tests/test_transform.py

```diff
@@ -75,3 +75,9 @@ def test_svg_figure_writes_width_height_and_view_box():
     assert 'width="400.0mm"' in written_content
     assert 'height="300.0mm"' in written_content
     assert 'viewBox="0 0 400.0 300.0"' in written_content
+
+def test_svg_figure__width_height_tostr():
+
```

Comment:
While black seems okay with the blank link after the `def` line (that I would personally drop), it wants two(!?) blank lines before the `def` line. This patch makes black happy:

```diff
--- a/tests/test_transform.py
+++ b/tests/test_transform.py
@@ -76,8 +76,8 @@ def test_svg_figure_writes_width_height_and_view_box():
     assert 'height="300.0mm"' in written_content
     assert 'viewBox="0 0 400.0 300.0"' in written_content
 
-def test_svg_figure__width_height_tostr():
 
+def test_svg_figure__width_height_tostr():
     svg_fig = transform.SVGFigure("400px", "300px")
     assert b'height="300.0px"' in svg_fig.to_str()
     assert b'width="400.0px"' in svg_fig.to_str()
```

I have created a pull request adding `--show-diff-on-failure` to the pre-commit CI now so that black errors will explain themselves in the CI log, in the future, at #105.

Reply:
done, thanks!